tests/test_precompiled.py

```diff
@@ -558,6 +561,20 @@ def test_sanitise_precompiled_letter_with_bad_address_returns_400(
     }
 
 
+@pytest.mark.parametrize('file', [
+    valid_letter,
+    pdf_with_no_metadata,
+], ids=['valid_letter', 'pdf_with_no_metadata'])
+def test_log_metadata_for_letter(
+    client,
+    file,
+    mocker,
+):
+    logger = mocker.patch('app.precompiled.current_app.logger.info')
+    log_metadata_for_letter(BytesIO(file), 'filename')
+    assert logger.called
```

Comment:
would we like to assert that it's called with a correct message?

Reply:
That would be more effort, but the log isn't critical: just for analysis.

I've tested it works manually, so I'm satisfied it's been sufficiently tested for the time being.

If it does break in future, we can always make the test more prescriptive.

Reply:
ok 👍🏼